app/src/components/AnalysisPane.js

```diff
@@ -1,147 +1,194 @@
+          bg='white'
+          px={4}
+          alignItems='center'
+        >
+          {attrs.map(([key, value]) => (
+            <React.Fragment key={key}>
+              {result.Text} {value ? '|' : null} {value}
+            </React.Fragment>
+          ))}
+        </Flex>
+
+        {closeIcon}
+      </Flex>
+    );
+  }
 
+  const concepts = [...(result.ICD10CMConcepts ? result.ICD10CMConcepts : result.RxNormConcepts)];
+  concepts.sort(function (concept1, concept2) {
+    return concept2.Score - concept1.Score;
+  });
 
   return (
-    <div className={cs(s.result, isExcluded && s.excluded)}>
-      <button className={s.removeButton} onClick={onHide} aria-label="Exclude this item from the export" />
-      <h4>{result.Text}</h4>
-
-      <dl>
-        {attrs.map(([ key, value ]) => (
-          <>
-            <dt>{key}</dt>
-            <dd>{value}</dd>
-          </>
+    <Flex width='100%' alignItems='center'>
+      <Select
+        mr={2}
+        border='1px solid'
+        borderColor={concepts[0].Score < 0.5 ? 'red' : 'grey'}
+        borderRadius='0'
+        bg='white'
+        _hover={{ borderColor: concepts[0].Score < 0.5 ? 'red' : 'grey', boxShadow: 'none' }}
+      >
+        {concepts.map((concept) => (
+          <option key={concept.Code} value={concept.Code}>
+            {result.Text} {concept.Code ? ' | ' : null} {concept.Code} {concept.Description ? ' | ' : null}{' '}
+            {concept.Description} &nbsp;&nbsp; {(concept.Score * 100).toPrecision(4)}%
+          </option>
         ))}
-      </dl>
-    </div>
-  )
-}
+      </Select>
 
-function ResultTable({
-  results,
-  category,
-  onToggleItem,
-  excludedItems
-}) {
-  const filteredResults = useMemo(() => results.filter(r => r.Category === category), [ results, category ]);
-  const [ open, setOpen ] = useState(false);
-
-
-  return <div className={s.resultTable}>
-    <h3 className={open ? s.expanded : s.contracted} onClick={() => setOpen(x => !x)}>{displayNames[category]}</h3>
-
-    <HeightSlider>
-      {open ?
-        <div>
-          {filteredResults.map((r, i) => (
-            <ResultRow
-              result={r}
-              key={r.id}
-              onToggleItem={onToggleItem}
-              excludedItems={excludedItems}
-            />
-          ))}
-        </div>
-      : null }
-    </HeightSlider>
-  </div>
+      {closeIcon}
+    </Flex>
+  );
 }
 
+function ResultTable({ results, category, onToggleItem, excludedItems, onResultDelete, onResultAdd }) {
+  const filteredResults = useMemo(() => results.filter((r) => r.Category === category), [results, category]);
+  const [inputValue, setInputValue] = React.useState('');
+  const handleInputChange = (event) => setInputValue(event.target.value);
+
+  const addIcon = (
+    <IconButton
+      aria-label='Add'
+      icon={<AddIcon />}
+      onClick={() => [inputValue.trim() !== '' ? onResultAdd(inputValue, category) : null, setInputValue('')]}
```

Comment:
We should call `onResultAdd` with the trimmed value as a better UX. Also, the input should only be cleared when the result was added. Here, I could add a bunch of whitespace and it would cleared even if we decided not to add.

Reply:
Good point.